packages/taquito/test/helpers.ts

```diff
@@ -200,3 +217,32 @@ export class RegisterGlobalConstantOperationBuilder {
     };
   }
 }
+
+export class RollupOriginationOperationBuilder {
```

Comment:
maybe we could still add the `Tx` as a prefix of the classname, since we might have SC rollups in the future as well? Just to remove possible confusion between the 2 rollups